### PR TITLE
devil: GCC 5 fix from NetBSD

### PR DESCRIPTION
**Pull request by ilovezfs**
Also, re-enable bottles.

### CODE REVIEW

File: Formula/devil.rb [hunk 1]
```diff
@@ -47,26 +47,35 @@ class Devil < Formula
...
     args = %W[
       --disable-debug
       --disable-dependency-tracking
       --prefix=#{prefix}
       --enable-ILU
     ]
     args << "--enable-ILUT" if build.stable? || build.with?("ilut")
+    args << "--disable-tiff" if build.without? "libtiff"
+
+    # Fixes the test for HEAD builds without the ilut option
```
Comment (dunn) [inline Formula/devil.rb]: Has this also been reported?

**Comment by ilovezfs** [comment]
@dunn now it has :smiley:

**Comment by dunn** [comment]
🙆

**Comment by apjanke** [comment]
:+1:

**Pull request closed without merging**